docs/index.rst

```diff
@@ -3,10 +3,30 @@ DefectDojo's Documentation
 
```

Comment:
No comment on this. Please just make sure the image paths are specified correctly. Thanks!